### PR TITLE
bugfix/issues-24-27-28

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**Are you a current Fivetran customer?** 

Fivetran employee

**What change(s) does this PR introduce?** 

Edit after

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide explanation how the change is non breaking below.)

**Is this PR in response to a previously created Issue**

- [x] Yes, Issue [link issue number here]
- [ ] No 
#24 
#27 
#28 

**How did you test the PR changes?** 

 
- [x] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# dbt_stripe v0.6.1
+## 🐞 Bug Fixes 🐞
+- [#25](https://github.com/fivetran/dbt_stripe/issues/25): Updating docs to add `dbt_stripe` documentation in addition to `dbt_stripe_source` documentation.
```
Comment (fivetran-jamie) [inline CHANGELOG.md:3]:
```suggestion
- [#24](https://github.com/fivetran/dbt_stripe/issues/24): Updating docs to add `dbt_stripe` documentation in addition to `dbt_stripe_source` documentation.
```

24 right?

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ This package contains transformation models, designed to work simultaneously wit
...
 | [stripe__subscription_line_items](https://github.com/fivetran/dbt_stripe/blob/master/models/stripe__subscription_line_items.sql) | Each record represents a subscription invoice line item, enriched with details about the associated charge, customer, subscription, and plan. Use this table as the starting point for your company-specific churn and MRR calculations. |
-| [stripe__customer_overview](https://github.com/fivetran/dbt_stripe/blob/master/models/stripe__customer_overview.sql)       | Each record represents a customer, enriched with metrics about their associated transactions.                                                                                                                                            |
+| [stripe__customer_overview](https://github.com/fivetran/dbt_stripe/blob/master/models/stripe__customer_overview.sql)       | Each record represents a customer, enriched with metrics about their associated transactions.  Transactions with no customer description will have a customer description of "No associated customer".                                                                                                                                          |
```
Comment (fivetran-jamie) [inline README.md:18]:
```suggestion
| [stripe__customer_overview](https://github.com/fivetran/dbt_stripe/blob/master/models/stripe__customer_overview.sql)       | Each record represents a customer, enriched with metrics about their associated transactions.  Transactions with no associated customer will have a customer description of "No associated customer".                                                                                                                                          |
```

**Review by fivetran-jamie — COMMENTED** [review]
looks mostly good -- proposed little tweaks + there's the question about the row counts in hex

File: models/stripe__customer_overview.sql [hunk 3]
```diff
@@ -137,8 +137,8 @@ select
     {% endfor %}
   {% endif %}
   
-from customer
-left join transactions_grouped 
+from transactions_grouped
+left join customer
```
Comment (fivetran-jamie) [inline models/stripe__customer_overview.sql:141]: given this is a `customer_overview` model i personally would expect transaction-less customers to be in here, but with 0s for their transaction metrics. so on top of customer-less transactions we should prob cover this case too. a full outer join may do the job

File: models/stripe__customer_overview.sql [hunk 4]
```diff
@@ -93,53 +93,109 @@ with balance_transaction_joined as (
...
+      {% if var('stripe__customer_metadata',[]) %}
+        {% for metadata in var('stripe__customer_metadata') %}
+            ,customer.{{ metadata }}
+        {% endfor %}
+      {% endif %}
+      
+    from transactions_grouped
+    left join customer 
+        on transactions_grouped.customer_id = customer.customer_id
+    where customer.customer_id is null or customer.description is null
```
Comment (fivetran-jamie) [inline models/stripe__customer_overview.sql:143]: why the `customer.description` part? given that we `coalesce(customer.description, customer.customer_id) as customer_description` i would imagine that customers can have null descriptions?  i think this is causing a duplicate with a customer who has an id but not a description, so they're included in both CTEs

Comment (fivetran-sheringuyen) [reply models/stripe__customer_overview.sql]: Meant to update to "and", should work now!

**Pull request merged**